checkov/main.py

```diff
@@ -353,6 +353,9 @@ def run(banner: str = checkov_banner, argv: List[str] = sys.argv[1:]) -> Optiona
             file = config.file
             scan_reports = runner_registry.run(root_folder=root_folder, external_checks_dir=external_checks_dir,
                                                files=file)
+            was_error = any([scan_report.error_status for scan_report in scan_reports])
+            if was_error:
```

Comment:
```suggestion
            is_error = any([scan_report.error_status for scan_report in scan_reports])
            if is_error:
```
Just a matter of code practices, doesn't really matter

Reply:
sure! will be fixed